package.json

```diff
@@ -6,22 +6,30 @@
     "scripts": {
         "build": "tsc",
         "test": "jest",
-        "start": "node ./dist/index"
+        "start": "node ./dist/index",
+        "dev": "ts-node-dev --respawn src/index.ts",
+        "format": "prettier ./ -w"
```

Comment:
this fixes the formatting of all the files. you can run a format on one file from the VSC command palette too by doing `cmd + p, >, Format Document With..., prettier`.